### ISSUE TITLE
Permission Error in Accessing Workspace

### ISSUE THREAD

**Issue by jenrogers1207** [multinet-app/multinet-girder#63]
After starting the client server the workspaces for my girder user account does not appear and I get the following error message: 
```
Executing GraphQL Request
request: query {
      workspaces { name }
    }
An error occurred while resolving field Query.workspaces
Traceback (most recent call last):
  File "/Users/jen/.local/share/virtualenvs/multinet-girder-NQjY1GFC/lib/python3.7/site-packages/graphql/execution/executor.py", line 450, in resolve_or_error
    return executor.execute(resolve_fn, source, info, **args)
  File "/Users/jen/.local/share/virtualenvs/multinet-girder-NQjY1GFC/lib/python3.7/site-packages/graphql/execution/executors/sync.py", line 16, in execute
    return fn(*args, **kwargs)
  File "/Users/jen/Documents/Github/multinet-girder/multinet/multinet/resolvers/query.py", line 7, in workspaces
    return [workspace for workspace in db.get_workspaces(name) if not name or workspace == name]
  File "/Users/jen/Documents/Github/multinet-girder/multinet/multinet/db.py", line 13, in wrapper
    return fun(*args, **kwargs)
  File "/Users/jen/Documents/Github/multinet-girder/multinet/multinet/db.py", line 48, in get_workspaces
    workspaces = sys.databases()
  File "/Users/jen/.local/share/virtualenvs/multinet-girder-NQjY1GFC/lib/python3.7/site-packages/arango/database.py", line 700, in databases
    return self._execute(request, response_handler)
  File "/Users/jen/.local/share/virtualenvs/multinet-girder-NQjY1GFC/lib/python3.7/site-packages/arango/api.py", line 58, in _execute
    return self._executor.execute(request, response_handler)
  File "/Users/jen/.local/share/virtualenvs/multinet-girder-NQjY1GFC/lib/python3.7/site-packages/arango/executor.py", line 81, in execute
    return response_handler(response)
  File "/Users/jen/.local/share/virtualenvs/multinet-girder-NQjY1GFC/lib/python3.7/site-packages/arango/database.py", line 697, in response_handler
    raise DatabaseListError(resp, request)
arango.exceptions.DatabaseListError: [HTTP 401][ERR 11] not authorized to execute this request
Errors in request: 1
[HTTP 401][ERR 11] not authorized to execute this request
127.0.0.1 - - [17/Jun/2019:12:03:51] "POST /api/v1/multinet/graphql HTTP/1.1" 200 87 "http://localhost:8080/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.90 Safari/537.36"
```

Along with this - error message from the google chrome devtool console:
<img width="302" alt="Screen Shot 2019-06-17 at 12 04 29 PM" src="https://user-images.githubusercontent.com/22663748/59630360-f8fb8380-90f9-11e9-82bf-21729937e951.png">

I am not providing a password as I assume it would use default password. I get no other errors starting the process aside from the above. 
The girder portal shows no workspaces under my account.

Things I have tried so far:
retrying the set up process
logging out of the girder portal and logging in 
providing the password with yarn:serve

**Comment by waxlamp** [comment]
I think this is a problem of data persistence.

The very first time you launch the Docker container for Arango (using the `docker-compose` step in the instructions), whatever password is specified--whether the default one, or one you supply--gets baked into the newly created Arango database data. Every subsequent time you launch the container, that password will be used (even if you continue supplying a password through the environment variable).

So, if you try again but launch Girder using the correct password (again, through the environment variable) then it should work. This is ultimately a problem of deployment, so hopefully this hiccup (which I faced myself a few days ago) will sort of go away now that we've shifted how the environment variables and defaults work.
